### PR TITLE
Move NEI handlers to new recipemap methods

### PR DESCRIPTION
**Pull request by miozune**

### CODE REVIEW

File: repositories.gradle [hunk 1]
```diff
@@ -1,6 +1,7 @@
 // Add any additional repositories for your dependencies here
 
 repositories {
+    mavenLocal()
```
Comment (Glease) [inline repositories.gradle:4]: ?

**Review by Glease — COMMENTED** [review]
not sure about mavenLocal(), but otherwise is good to merge

File: repositories.gradle (hunk 1 above)
Comment (miozune) [reply repositories.gradle:4]: Eh at least it doesn't cause crash on workflow. I'm also just lazy to type it when I test something

Comment (Glease) [reply repositories.gradle:4]: I'll just pretend it doesn't exist then. it's not like I don't have this locally..

**Review by Glease — APPROVED** [review]

**Pull request merged**